R/rownames.R

```diff
@@ -65,6 +65,31 @@ rownames_to_column <- function(df, var = "rowname") {
   new_df
 }
 
```

Comment:
Ok, I'll have another go. Agree with your comments about the function itself, will make some changes!